hey/cli.py

```diff
@@ -74,3 +76,13 @@ def ask(
 
     result = answer(user_input, no_style)
     console.print(result)
+
+@app.command()
+def auth(
+    user_input: str, # This for unhandled exceptions that happen when user gives too many args.
```

Comment:
No need for `user_input` as the `hey auth` wouldn't work without specifying an argument. Line 82 should be removed.

Reply:
Based on your past review, this is exactly what you said to remove, because "it can easily be leaked and will appear in the history of the OS". So this should stay to avoid stupid errors.

Reply:
@sepehrrasooli, I see some issues with that being there. Otherwise, there is nothing to be worried about. I tested it without that `user_input` thing and worked as expected. That `user_input` forces users to run such a command to set their token:

```
hey auth SOMETHING
Enter your token:
```

But if you try to run the following one, it raises an issue as you haven't valued the `user_input` parameter.

```
hey auth
Error..
```

Removing the `user_input` will fix `hey auth`.

Reply:
Now i get what you're saying, I'm very sorry about not testing this case more.
Thanks !

Reply:
You got this. Don't mention it. :D